oracle/build/dbimage/buildspec.yml

```diff
@@ -0,0 +1,17 @@
+version: 0.2
```

Comment:
Can we place this file  under dbimage/**aws** instead?

Reply:
Sure. I have added -j parameter to the zip command to exclude paths so we can create the zipfile as it was before this change